torch/csrc/jit/mobile/flatbuffer_loader.cpp

```diff
@@ -172,6 +173,12 @@ mobile::Module FlatbufferLoader::parseModule(
   }
   IValue& module_ivalue = getIValue(module->state_obj());
 
+  auto extra_files_offsets = module_->extra_files();
```

Comment:
Let's pull this out as a separate function. i.e. parseModule should only parseModule; if you want extra files call a separater function that returns the file content.